projects/fsastorefrontlib/src/cms-components/checkout/components/cart-coupon/cart-coupon.component.html

```diff
@@ -1,10 +1,5 @@
 <ng-container *ngIf="(isCartStable$ | async) && (cart$ | async) as cart">
-  <ng-container
-    *ngIf="
-      cart.insuranceQuote?.state?.code !== 'BIND' &&
-      (userRegistered$ | async) === 'current'
```

Comment:
Could we also remove userRegistered variable from component?

Reply:
It's already deleted from component